extensions/npm/src/tasks.ts

```diff
@@ -70,7 +70,7 @@ export class NpmTaskProvider implements TaskProvider {
 			} else {
 				packageJsonUri = _task.scope.uri.with({ path: _task.scope.uri.path + '/package.json' });
 			}
-			return createTask(this.context, kind, `${kind.script === INSTALL_SCRIPT ? '' : 'run '}${kind.script}`, _task.scope, packageJsonUri);
+			return createTask(this.context, kind, `${kind.script === INSTALL_SCRIPT ? '' : 'run '}"${kind.script}"`, _task.scope, packageJsonUri);
```

Comment:
Instead of just quoting every script, I would prefer that we only quote scripts that have a space in them. No need to add quotes when they aren't needed.

Reply:
To reduce the number of places we need to "remember" to add quotes, we can just add them in one place where we create the command line: 
https://github.com/microsoft/vscode/blob/59ba6494b9eb3e13a7cb558a06e3018a74f30af1/extensions/npm/src/tasks.ts#L318-L318